app/modules/eth/transactions/transactions.route.ts

```diff
@@ -45,14 +47,16 @@ class AppsRoute implements IRoute {
       {
         method: HttpMethods.POST,
         auth: AuthStrategies.OAUTH,
+        middlewares: [this.authDeviceCheckMiddleware],
```

Comment:
Can't we just pass the class in this array and create the instance in the config? As it'd be a little bit tedious because we know a middleware will only have 1 method. `init()`
```javascript
middlewares: [AuthDeviceCheckMiddleware]
```

Reply:
Still having trouble with this. `typeof AuthDeviceCheckMiddleware` returns a function. How do we pass it to an interface for type checking? Is it necessary to have it initialized in `restify.ts`?